charts/velero/crds/backuprepositories.yaml

```diff
@@ -8,20 +8,23 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.7.0
   creationTimestamp: null
-  name: resticrepositories.velero.io
+  name: backuprepositories.velero.io
```

Comment:
What if the existing cluster contains the resticrepositories CRs? Can helm remove the resticrepositories CRDs if there are resticrepositories CRs within the cluster?

Reply:
We wouldn't remove existing CRs when upgrading to a higher version, but user could not using `velero restic repo get` to list the resticrepositories.

Reply:
Since the CRD changes from `resticrepositories.velero.io` to `backuprepositories.velero.io`.
If I remember correctly, helm upgrade will
- install a new CRD `backuprepositories.velero.io`
- remove the CRD `resticrepositories.velero.io`

Therefore, is there were exiting resticrepositories CRs within the cluster, I _guess_ the helm upgrade would fail because the CRD `resticrepostiories.velero.io` can't be removed successfully (because there are resticrepositories CRs existed).

Reply:
I've tested that it will still keep resticrepositories.velero.io CRD and CRs after successfully upgrading, also backuprepositories.velero.io could be created